superset/dashboards/commands/importers/v1/utils.py

```diff
@@ -115,7 +115,7 @@ def import_dashboard(
     # TODO (betodealmeida): move this logic to import_from_dict
     config = config.copy()
     for key, new_name in JSON_KEYS.items():
-        if config.get(key):
+        if config.get(key) is not None:
```

Comment:
is `if key in config:` enough here?

basically, is the `is not None` test checking if `config.get` is returning the default value, or if None is set in the config for that key?

Reply:
Right, I was thinking of the case where we have they key and it's set to `None`. In that case we might as well skip the serialization and simply ignore it.